DEVELOPING.md

```diff
@@ -0,0 +1,12 @@
+# Developing Trow
+
+The easiest way to build and test Trow is by using the [Dockerfiles](./docker/README.md).
+
+Trow is written in [Rust](https://www.rust-lang.org/) and can be compiled, run and tested with the
```

Comment:
Similar to above comments, I think a Hermetic local environment, for test/build in Docker could be useful.